### PR TITLE
Add error logs

### PR DESCRIPTION
**Pull request by dongli2015**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -90,8 +90,7 @@ lazy val sharedReleaseProcess = Seq(
     publishArtifacts,
     setNextVersion,
     commitNextVersion,
-    ReleaseStep(action = Command.process("sonatypeReleaseAll", _), enableCrossBuild = true)
-    // pushChanges
+    pushChanges
```
Comment (dongli2015) [inline build.sbt]: the last step of sbtrelease is to push the changes to master branch.

File: build.sbt [hunk 2]
```diff
@@ -298,6 +285,7 @@ lazy val core = Project(
   	  D.akkaActor,
   	  D.akkaPersistence,
       D.akkaPersistenceQuery,
+      D.akkaSlf4j,
   	  D.akkaTest % "test",
   	  D.scalaTest % "test",
       D.akkaPersistenceInMemory % "test"
```
Comment (abalonperin) [inline build.sbt]: Can you vertically align all dependencies?

File: akka-d3-core/src/main/scala/akka/contrib/d3/protobuf/D3MessageSerializer.scala [hunk 3]
```diff
@@ -36,6 +39,8 @@ private[d3] class D3MessageSerializer(val system: ExtendedActorSystem)
     extends SerializerWithStringManifest with BaseSerializer {
   import D3MessageSerializer._
 
+  val logger = LoggerFactory.getLogger(getClass.getName)
```
Comment (stefansperber) [inline akka-d3-core/src/main/scala/akka/contrib/d3/protobuf/D3MessageSerializer.scala]: private

File: akka-d3-core/src/main/scala/akka/contrib/d3/protobuf/D3MessageSerializer.scala [hunk 4]
```diff
@@ -45,6 +50,16 @@ private[d3] class D3MessageSerializer(val system: ExtendedActorSystem)
     ser
   }
 
+  private def deserialize(bytes: Array[Byte], serializerId: Int, manifest: String): Option[AnyRef] = {
+    serialization.deserialize(bytes, serializerId, manifest) match {
+      case Failure(exception) ⇒
```
Comment (stefansperber) [inline akka-d3-core/src/main/scala/akka/contrib/d3/protobuf/D3MessageSerializer.scala]: If this is an exception then you can avoid a lot of uncertainty by just putting the exception at the right place of the `logger.error`. Will make sure the whole stacktrace and message is logged.

File: build.sbt [hunk 5]
```diff
@@ -282,7 +270,6 @@ lazy val docs = Project(
   )
   .settings(moduleName := "docs")
   .settings(d3Settings)
-  .settings(micrositesSettings)
```
Comment (stefansperber) [inline build.sbt]: There's probably also a plugin that you can delete now.

File: build.sbt (hunk 1 above)
Comment (stefansperber) [inline build.sbt]: In `core-merchants` we use these steps and they work well (also with cross compilation):
```
releaseProcess := Seq[ReleaseStep](
    checkSnapshotDependencies,
    inquireVersions,
    runClean,
    releaseStepCommandAndRemaining("+test"),
    setReleaseVersion,
    commitReleaseVersion,
    tagRelease,
    releaseStepCommandAndRemaining("+publish"),
    setNextVersion,
    commitNextVersion,
    pushChanges
  )
```

File: akka-d3-core/src/main/scala/akka/contrib/d3/protobuf/D3MessageSerializer.scala [hunk 6]
```diff
@@ -45,6 +50,16 @@ private[d3] class D3MessageSerializer(val system: ExtendedActorSystem)
...
+        logger.error(s"Could not deserialize bytes: $bytes, serializerId: $serializerId, manifest: $manifest. Error: ${exception.getStackTrace.toSeq.mkString("\n")}")
```
[the remark this answers is not in the document]
Comment (dongli2015) [reply akka-d3-core/src/main/scala/akka/contrib/d3/protobuf/D3MessageSerializer.scala]: it's at the end 
```
Error: ${exception.getStackTrace.toSeq.mkString("\n")
```

File: build.sbt (hunk 1 above)
Comment (dongli2015) [reply build.sbt]: @stefansperber thank you! 

@abalonperin 
For `releaseStepCommand("build")`:
it's mentioned here https://gitter.im/sbt/sbt/archives/2017/05/02
```
If you only use releaseStepCommand, it is not going to execute all of the instructions that really are taking place under the hood when doing these cross operations -- just the first -- which is definitely not what you want. So, that's why you use releaseStepCommandAndRemaining: execute all, and any subsequent instructions, until completion.
```

From the code it's clear 
https://github.com/sbt/sbt-release/blob/master/src/main/scala/ReleasePlugin.scala#L92~L117

So I'll update `releaseStepCommand("build")` to `releaseStepCommandAndRemaining("+build")`.

For `publishArtifacts`, `enableCrossBuild=true`, i think it works same as `releaseStepCommandAndRemaining("+publish")`.

Comment (stefansperber) [reply build.sbt]: With the release steps that I posted you can then even just run `release:defaults` (or similar – ask me later) and it'll use/update the correct version + cross compile if needed.

**Pull request merged**